ndsl/stencils/__init__.py

```diff
@@ -1 +1,21 @@
+from .c2l_ord import CubedToLatLon
+from .corners import CopyCorners, CopyCornersXY, FillCornersBGrid
+from .testing.grid import Grid  # type: ignore
+from .testing.parallel_translate import (
+    ParallelTranslate,
+    ParallelTranslate2Py,
+    ParallelTranslate2PyState,
+    ParallelTranslateBaseSlicing,
+    ParallelTranslateGrid,
+)
+from .testing.savepoint import SavepointCase, Translate, dataset_to_dict
+from .testing.temporaries import assert_same_temporaries, copy_temporaries
+from .testing.translate import (
+    TranslateFortranData2Py,
+    TranslateGrid,
+    pad_field_in_j,
+    read_serialized_data,
+)
```

Comment:
I think the testing stuff could still be in stencils.testing but that's not a big deal either way

Reply:
Move down to `testing` it's specific enough